### ISSUE TITLE
MixCloud ExtractorError: Failed to parse JSON

### ISSUE THREAD

**Issue by Pitel** [rg3/youtube-dl#13885]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.08.09*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2017.08.09**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add the `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

```
[debug] System config: []
[debug] User config: [u'-v', u'--console-title']
[debug] Custom config: []
[debug] Command-line args: [u'https://www.mixcloud.com/VektorR1/vektor-40-210617']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2017.08.09
[debug] Python version 2.7.13 - Linux-4.10.0-30-generic-x86_64-with-Ubuntu-17.04-zesty
[debug] exe versions: ffmpeg 3.2.4-1build2, ffprobe 3.2.4-1build2
[debug] Proxy map: {}
```

---

### Description of your *issue*, suggested solution and other information

Explanation of your *issue* in arbitrary form goes here. Please make sure the [description is worded well enough to be understood](https://github.com/rg3/youtube-dl#is-the-description-of-the-issue-itself-sufficient). Provide as much context and examples as possible.
If work on your *issue* requires account credentials please provide them or explain how one can obtain them.

```
youtube-dl 'https://www.mixcloud.com/VektorR1/vektor-40-210617'                                                                                                        1 ↵ 
[debug] System config: []
[debug] User config: [u'-v', u'--console-title']
[debug] Custom config: []
[debug] Command-line args: [u'https://www.mixcloud.com/VektorR1/vektor-40-210617']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2017.08.09
[debug] Python version 2.7.13 - Linux-4.10.0-30-generic-x86_64-with-Ubuntu-17.04-zesty
[debug] exe versions: ffmpeg 3.2.4-1build2, ffprobe 3.2.4-1build2
[debug] Proxy map: {}
[mixcloud] VektorR1-vektor-40-210617: Downloading webpage
ERROR: VektorR1-vektor-40-210617: Failed to parse JSON  (caused by ValueError('No JSON object could be decoded',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/home/pitel/bin/youtube-dl/youtube_dl/extractor/common.py", line 676, in _parse_json
    return json.loads(json_string)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
Traceback (most recent call last):
  File "/home/pitel/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 776, in extract_info
    ie_result = ie.extract(url)
  File "/home/pitel/bin/youtube-dl/youtube_dl/extractor/common.py", line 433, in extract
    ie_result = self._real_extract(url)
  File "/home/pitel/bin/youtube-dl/youtube_dl/extractor/mixcloud.py", line 89, in _real_extract
    play_info = self._decrypt_play_info(encrypted_play_info, track_id)
  File "/home/pitel/bin/youtube-dl/youtube_dl/extractor/mixcloud.py", line 69, in _decrypt_play_info
    video_id)
  File "/home/pitel/bin/youtube-dl/youtube_dl/extractor/common.py", line 680, in _parse_json
    raise ExtractorError(errmsg, cause=ve)
ExtractorError: VektorR1-vektor-40-210617: Failed to parse JSON  (caused by ValueError('No JSON object could be decoded',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by darnel** [comment]
Duplicates #13867

**Comment by PSlava** [comment]
No. I confirm, it's a new issues.

**Comment by AndyRubio** [comment]
This is the second change to the decrypt key this week. Are they onto us?

**Comment by dmpe** [comment]
Did they changed it again? The key (as of 14.8) is still `pleasedontdownloadourmusictheartistswontgetpaid`.

Should I open a new bug ?
Thanks.
```
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/common.py", line 676, in _parse_json
    return json.loads(json_string)
  File "/usr/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/common.py", line 676, in _parse_json
    return json.loads(json_string)
  File "/usr/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/YoutubeDL.py", line 776, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/common.py", line 433, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/mixcloud.py", line 90, in _real_extract
    play_info = self._decrypt_play_info(encrypted_play_info, track_id)
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/mixcloud.py", line 70, in _decrypt_play_info
    video_id)
  File "/usr/local/lib/python3.5/dist-packages/youtube_dl/extractor/common.py", line 680, in _parse_json
    raise ExtractorError(errmsg, cause=ve)
youtube_dl.utils.ExtractorError: NTSRadio-radio-ghibli-part-3-2002-2014-17th-april-2017: Failed to parse JSON  (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)',)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by AndyRubio** [comment]
Same for me, open a new ticket.

**Comment by dmpe** [comment]
hmm, just tried again updating youtube-dl and then again above link and still getting errors. Will file new bug.

**Comment by AndyRubio** [comment]
It's fixed but not yet released (a20951a57bddd4a0102cd776ff93a2adc6db77d).
Wait til new release before submitting bug.

**Comment by dmpe** [comment]
Aha, ok. thanks. Will wait.
